packages/turf-interpolate/test.js

```diff
@@ -0,0 +1,58 @@
+const test = require('tape');
+const fs = require('fs');
+const path = require('path');
+const load = require('load-json-file');
+const write = require('write-json-file');
+const round = require('@turf/helpers').round;
+const featureEach = require('@turf/meta').featureEach;
+const chromatism = require('chromatism');
+const interpolate = require('./');
+
+const directories = {
+    in: path.join(__dirname, 'test', 'in') + path.sep,
+    out: path.join(__dirname, 'test', 'out') + path.sep
+};
+
+const fixtures = fs.readdirSync(directories.in).map(filename => {
+    return {
+        filename,
+        name: path.parse(filename).name,
+        geojson: load.sync(directories.in + filename)
+    };
+});
+
+test('turf-interpolate', t => {
+    for (const {filename, name, geojson}  of fixtures) {
+        const {property, cellSize, units, weight} = geojson.properties;
+
+        const grid = interpolate(geojson, cellSize, property, units, weight);
+        const result = colorize(grid, property);
```

Comment:
Added `@turf/truncate` to results (reduce coordinates to 6 decimals).

Node 7 & 8 were passing, but Node 6 was failing (0.0000001 of a difference).